department-of-veterans-affairs/abd-vro
# 1777 coverage increase

## What was the problem?
Not enough coverage.

Associated tickets or Slack threads:

Solves this part of story 1777
"Test coverage meets jacoco_minimum_coverage of 0.5 (commit https://github.com/department-of-veterans-affairs/abd-vro/commit/9f1d10d24bd5c43a628f6236e67d6ed1c224ab03)"

## How does this fix it?[^1]
Unused untested code was removed.

## How to test this PR
./gradlew :svc-bip-api:test   


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
